supplementary_style_guide/glossary_terms_conventions/product_conventions/amq7.adoc

```diff
@@ -1,7 +1,6 @@
 [[red-hat-jboss-amq7-conventions]]
 
-
-Responsible person: Ben Hardesty (bhardest@redhat.com)
+For information on official product naming, including long- and short-form names, please contact naming@redhat.com.
```

Comment:
Since using "please" is against IBM style guide, would you mind removing all instances in this PR?

Reply:
Also, the listinfo for naming@redhat.com [1] says that it's an internal list. Just checking whether or not this is something that should be shared publicly.

[1] http://post-office.corp.redhat.com/mailman/listinfo/naming

Reply:
Hey @bergerhoffer 

Azure - I do believe we should evaluate this entire section and define our guidance on third party products. Les was working on something related to this so I'll check with him on status. 

Naming - let me check in with them. I'm of the opinion we shed naming concerns entirely.

Reply:
@cbudz  and @bergerhoffer , I think it's a good idea to define how we refer to all the major cloud providers. For instance, we start off with Microsoft Azure, but we drop the Microsoft in succeeding mentions unless Microsoft Azure is a UI label or some other circumstance that requires the company name in front of the product.

We spell out Amazon Web Services at the beginning of a title or article, but we use AWS almost exclusively.

We really haven't done enough with Google Cloud Platform (GCP) or IBM Cloud to have developed inconsistent citations.

Reply:
Thanks @leswilliams44. What do you think about at least adding some sort of caveat at the top? Something like how this is Red Hat's guidance on referring to the Azure products, but for the latest, see the official Azure documentation, or something?

Reply:
@bergerhoffer, that sounds reasonable. I'll take a stab at it.